### DISCUSSION TITLE
Color skinning for Synesthesia mod?

### DISCUSSION THREAD

**Discussion by Moi-a** [ppy/osu#26672] [Ideas] [answered]
The colors are rather dark and hard to see for someone used to lighter colored skins, and they are not very appealing shades of colors either.
P.S. - I dont use this site so I hope this isn't the wrong place for this.

**Comment by nagi-desuuu** [comment]
Isn't this also going to affect mania's Timing-based coloring, though?

**Comment by Rekunan** [comment, marked as answer]
Duplicate of: #24036
